app/controllers/application_controller.rb

```diff
@@ -9,7 +9,7 @@ class ApplicationController < ActionController::Base
   before_action :set_sentry_user, except: :check, unless: :devise_controller?
 
   def check
-    render json: { status: "OK", version: release_version, sha: ENV["SHA"], environment: Rails.env }, status: :ok
+    head :ok
```

Comment:
Is this what paas uses to decide whether to let our app live for another few moments?

Reply:
Yep, hence the simple, speedy response. "Please don't kill me!"